nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/magick/utils/ImageResizer.java

```diff
@@ -56,6 +62,11 @@ public static ImageInfo resize(String inputFile, String outputFile, int targetWi
         // hack to manage jpeg default background
         if (outputFile.endsWith(JPEG_CONVERSATION_FORMAT)) {
             commandName = "jpegResizer";
+            Point size = scaleToMax(targetWidth, targetHeight, MAX_JEPG_DIMENSION);
```

Comment:
Do we have to create this huge target size (64k) each time ? this might impact image magic perf no?

Reply:
indeed this is done only if the requested target is bigger than 64k so :+1: